pages/api/mail/index.ts

```diff
@@ -0,0 +1,188 @@
+            <p style="font-size: 30px"><img src=https://images.squarespace-cdn.com/content/5e83092341f99d6d384777ef/1592547010897-WF00319AKLJCVGJZC3ZK/sdc+logo+with+name+alt.png?content-type=image%2Fpng
+            style="width: 250px; padding-bottom: 10px; color: #0c53a0" alt="SDC Logo"/></p>
+            <h2><b>Your class is starting in ${intervalHours} hours!</b></h2>
+            <h4>This is an automatic reminder for the following class:</h4>
+            <img src=${classes[i].imageLink} style="width: 68px; height: 77px;"/>
+            <p>Regards, Social Diversity for Children</p>
+            </body>
+            `;
+            mailerPromises.push(
+                send(
+                    process.env.EMAIL_FROM,
+                    parentRegs[j].parent.user.email,
+                    emailSubject,
+                    emailBody,
+                ),
+            );
+        }
+        // looping through volunteers that are registered to EACH class
+        for (let j = 0; j < volunteerRegs.length; ++j) {
+            const classStartDate = classes[i].startDate;
+            const emailSubject = `Class Reminder: ${
+                classes[i].name
+            } ${getWeekday(classes[i].weekday)}s ${getHours(
+                classStartDate,
+            ).slice(0, -2)}-${getHours(
+                classStartDate.setHours(classStartDate.getHours() + 1),
+            )}`;
+            const emailBody = `
+            <head>
+            <link
+                href="https://fonts.googleapis.com/css?family=Poppins"
+                rel="stylesheet"
+            />
+            <style>
+                body {
+                    font-family: "Poppins";
+                }
+            </style>
+            </head>
+            <body style="background-color: #fff; padding: 30px;">
+            <p style="font-size: 30px"><img src=https://images.squarespace-cdn.com/content/5e83092341f99d6d384777ef/1592547010897-WF00319AKLJCVGJZC3ZK/sdc+logo+with+name+alt.png?content-type=image%2Fpng
+            style="width: 250px; padding-bottom: 10px; color: #0c53a0" alt="SDC Logo"/></p>
+            <h2><b>Your class is starting in ${intervalHours} hours!</b></h2>
+            <h4>This is an automatic reminder for the following class:</h4>
+            <img src=${classes[i].imageLink} style="width: 68px; height: 77px;"/>
+            <p>Regards, Social Diversity for Children</p>
+            </body>
+            `;
+            mailerPromises.push(
+                send(
+                    process.env.EMAIL_FROM,
+                    volunteerRegs[j].volunteer.user.email,
+                    emailSubject,
+                    emailBody,
+                ),
+            );
+        }
+    }
+    return;
+}
+
+/**
+ * Helper function to get the hour of a date in 12 hour format
+ * @param d date object from DB
+ * @returns the start time of a class (hour)
+ */
+function getHours(d: Date): string {
```

Comment:
Should these two functions be in `utils`? They seem to be generic enough to be potentially used somewhere else too. Thoughts? @victoranyun